src/frontend/citizen-portal/src/app/components/dispute-ticket-stepper/dispute-ticket-stepper.component.ts

```diff
@@ -98,12 +98,22 @@ export class DisputeTicketStepperComponent implements OnInit, AfterViewInit {
     private formUtilsService: FormUtilsService,
     private translateService: TranslateService,
     private toastService: ToastService,
-    private config: ConfigService,
+    public config: ConfigService,
     private lookups: LookupsService
   ) {
     // config or static
     this.isMobile = this.utilsService.isMobile();
     this.defaultLanguage = this.translateService.getDefaultLang();
+
+    this.bcFound = this.config.provincesAndStates.filter(x => x.provAbbreviationCd === "BC");
+    this.canadaFound = this.config.countries.filter(x => x.ctryLongNm === "Canada");
+    this.usaFound = this.config.countries.filter(x => x.ctryLongNm === "USA");
```

Comment:
For those "found" should only have one result and only ID is needed, suggested declaring as follow
bcProvinceCode: ProvinceCodeValue;

let bcFound = this.config.provincesAndStates.filter(x => x.provAbbreviationCd === "BC");
bcProvinceCode = bcFound[0];

Reply:
a fairly good assumption just wanting to avoid crashing.  Also need the name and Id for dropdown.

Reply:
If(bcFound.length === 0)
...